addons/project/views/project_task_type_views.xml

```diff
@@ -55,7 +55,6 @@
                         <group>
                             <group>
                                 <field name="name"/>
-                                <field name="sequence" groups="base.group_no_one"/>
```

Comment:
What was this field doing, and why remove it?

Reply:
simply displaying the sequence of the stage :shrug: 
The specs specifies to only display the *name* and the *kanban_folded* fields, i'll check again with AJU to confirm this, thanks for the review :)

Reply:
I asked to Alexandra and she said to keep that field in that view (with the same visibility condition). 🙂